code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -85,10 +85,15 @@
 
 /obj/machinery/portable_atmospherics/canister/Initialize(mapload, datum/gas_mixture/existing_mixture)
 	. = ..()
+
 	if(existing_mixture)
 		air_contents.copy_from(existing_mixture)
 	else
 		create_gas()
+
+	var/random_quality = rand(0, 100) / 100
+	pressure_limit = initial(pressure_limit) * (1 + 0.1 * random_quality)
```

Comment:
Is this enough to have much effect?

Reply:
example: rand give me 44, so random quality 0.44
50000 * (1 + 0.1 * 0.44) = 52200
so the amount is not great, but is enough to be noticeable (of course if the rand behaves and gives you bigger numbers)

Reply:
I could go up to 20% more but if I do that I'm going to want to nerf the base pressure capacity, that sound better?

Reply:
I'd be alright with that, want to make sure it properly effects things

Reply:
If you nerf the base capacity make sure it doesn't force toxins to upgrade their tank in the worst case, since that's a jump in tech if I recall things correctly

Reply:
I'll nerf it so the average is roughly the same as now but if they get a really bad one they can just try a different canister

Reply:
The base pressure limit for the tier 1 canister can swing between the base amount of 46000 up to 55200 now, the other tiers were similarly adjusted.